### PR TITLE
feat: added improved camera setters & getters

### LINKED ISSUE #33: MapViewCamera Pitch & Direction in MapViewCameraOperations
We should be able to set the pitch and direction of the camera in the same way zoom can be set on https://github.com/Rallista/maplibre-compose-playground/blob/main/compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraOperations.kt

These functions can behave the same way as zoom and check for validity. We probably need a valid direction function too.

### PR DESCRIPTION
**Pull request by Archdoog**
- Adds getters for camera's zoom, pitch and direction.
- Adds `MapViewCamera.setPitch`.
- Adds `MapViewCamera.setDirection`.
- Adds direction validation.

Closes #33

### CODE REVIEW

File: compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraDirection.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+/**
+ * Set the camera's direction (0 to 360 degrees).
+ *
+ * This function will ensure that the direction is within the min and max direction values.
+ *
+ * ```kotlin
+ * val camera = remember { mutableStateOf(MapViewCamera()) }
```
Comment (ianthetechie) [inline compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraDirection.kt:28]: Don't we have a `rememberSaveableMapViewCamera` helper or something?

File: compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraPitch.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+/**
+ * Set the camera's pitch.
+ *
+ * This function will ensure that the pitch is within the min and max pitch values.
+ *
+ * ```kotlin
+ * val camera = remember { mutableStateOf(MapViewCamera()) }
```
Comment (ianthetechie) [inline compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraPitch.kt:25]: Ditto prev comment

**Review by ianthetechie — APPROVED** [review]
Looks good! Just one minor thing to clean up in the docs maybe.

File: compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraDirection.kt (hunk 1 above)
Comment (Archdoog) [reply compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraDirection.kt]: We sure do! Good call out. Updated 👍

File: compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraPitch.kt (hunk 2 above)
Comment (Archdoog) [reply compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraPitch.kt]: Updated 👍

**Pull request merged**